libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1855,6 +1865,16 @@ void AP_GPS::calc_blended_state(void)
     timing[GPS_BLENDED_INSTANCE].last_fix_time_ms = 0;
     timing[GPS_BLENDED_INSTANCE].last_message_time_ms = 0;
 
+    if (state[0].have_undulation) {
+        state[GPS_BLENDED_INSTANCE].have_undulation = true;
+        state[GPS_BLENDED_INSTANCE].undulation = state[0].undulation;
```

Comment:
The vertical position is possibly blended here, but the undulation isn't? Given the differences in stored table accuracies that will result in random step changes.

Reply:
only used in one plane, in AP_OpenDroneID and both receivers should be giving the same undulation. It is a very slowly changing number with distance. We really only support blending between two receivers of the same type due to the way we use the accuracy numbers